lib/src/presentation/view/characters_page.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import '../../core/util/strings_constants.dart';
+import '../../core/util/text_styles.dart';
+
+class CharactersPage extends StatefulWidget {
+  const CharactersPage({
+    required this.title,
+  });
+
+  final String title;
+
+  @override
+  _CharactersPageState createState() => _CharactersPageState();
+}
+
+class _CharactersPageState extends State<CharactersPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Container(
+          alignment: Alignment.center,
+          child: Text(
+            widget.title,
+          ),
+        ),
+        automaticallyImplyLeading: false,
```

Comment:
do you need to do this on this screen? check it on the rest of the pages

Reply:
done